autometa/__main__.py

```diff
@@ -184,14 +186,15 @@ def entrypoint():
         sys.exit(1)
     except Exception as err:
         issue_request = """
+        An error was encountered!
 
         Please help us fix your problem!
 
         You may file an issue with us at https://github.com/KwanLab/Autometa/issues/new/choose
         """
         err.issue_request = issue_request
         logger.exception(err)
-        logger.info(err.issue_request)
+        logger.info(issue_request)
```

Comment:
I think this should be followed by a `sys.exit(1)` to exit the pipeline once the error is caught and logged.